### ISSUE TITLE
keepalived: route not added in conf when next_hop is your default gateway

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5889]
**Describe the bug**
When you configure a L3 network as "Registration" and use IP address of your default gateway as next hop, there is no static routes added in `/usr/local/pf/var/conf/keepalived.conf`

**To Reproduce**
1. Configure your management interface with `dhcp` daemon
2. Create a L3 network as "Registration" with next hop value equals to default gateway of PacketFence server
3. Save

Check content of `/usr/local/pf/var/conf/keepalived.conf`:
```
[...]
static_routes {

}
```
**Expected behavior**
Route to L3 network should be present in `/usr/local/pf/var/conf/keepalived.conf`

**Comment by fdurand** [comment]
It looks that because next_hop is not in the management network.
So if PacketFence is not able to know what is the interface to use, then it bypass the route.

**Comment by nqb** [comment]
You're right. In this configuration, next hop is not in the management network but it matchs an existing interface defined in PF configuration. GUI doesn't complain when I added next hop. However, I know that is the case when you add a next hop that is not on any network attached to PF.

From my point of view, in this specific situation, route should be created in keepalive configuration.

**Comment by fdurand** [comment]
So the dhcp daemon needs to be added on the interface use to route the packet and it will work.

**Comment by nqb** [comment]
You're right, route to L3 network is present in `/usr/local/pf/var/conf/keepalived.conf` after I added `dhcp` daemon on this interface.

Perhaps we should add a new check when we declare a next hop to ensure its IP address matchs a PacketFence interface where a DHCP daemon is listening.
